samples/server/petstore/scala-cask/src/main/scala/mo/del/pack/age/ApiResponse.scala

```diff
@@ -0,0 +1,53 @@
+/**
```

Comment:
the  "mo/del" folder looks a bit weird to me.
should it be simply just "model"?

Reply:
I was just testing out weird/nested configs, but you're right. I've updated it now to be a more standard package structure